### PR TITLE
using libmatrix_malos

### PR DESCRIPTION
**Pull request by eljuguetero**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -48,15 +42,17 @@ set(malos_src
   driver_lirc.cpp driver_lirc.h
   driver_servo.cpp driver_servo.h
   driver_gpio.cpp driver_gpio.h
-  tcp_client.cpp tcp_client.h
```
Comment (arhuaco) [inline src/CMakeLists.txt:51]: Where did tcp_client go? It's used in the ZigBee driver.

**Review by arhuaco — APPROVED** [review]
Looks good. We are repeating this code in MALOS-eye.

File: src/CMakeLists.txt (hunk 1 above)
Comment (eljuguetero) [inline src/CMakeLists.txt]: Your alright,

Comment (eljuguetero) [reply src/CMakeLists.txt:51]: Your alright,
tcp_client.cpp tcp_client.h must be moved to matrix-malos-zigbee soon

**Pull request merged**